build-tools/xaprepare/README.md

```diff
@@ -0,0 +1,173 @@
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Xamarin.Android build preparation utility](#xamarinandroid-build-preparation-utility)
+    - [Introduction](#introduction)
+    - [Why?](#why)
+    - [Supported operating systems](#supported-operating-systems)
+    - [Prerequisites](#prerequisites)
+        - [All systems](#all-systems)
+        - [macOS and Linux](#macos-and-linux)
+        - [macOS](#macos)
+    - [Build configuration](#build-configuration)
+        - [File naming convention](#file-naming-convention)
+        - [Configuration directory](#configuration-directory)
+    - [Running](#running)
+        - [Scenarios](#scenarios)
+        - [Invocation](#invocation)
+        - [Log files](#log-files)
+
+<!-- markdown-toc end -->
+# Xamarin.Android build preparation utility
+
+## Introduction
+
+The task and purpose of this utility is to prepare the ``Xamarin.Android` source tree for build by 
```

Comment:
Too many backticks, screwing up formatting, and I assume you mean "`xamarin-android` source tree".

Reply:
I meant the product name - `Xamarin.Android` - not GitHub repo name `xamarin-android`